src/networks.json

```diff
@@ -1423,5 +1423,20 @@
     "explorer": "https://visionscan.org",
     "start": 75909,
     "logo": "ipfs://QmXgGmxDAW2Mheum3WX7Q52Zi9hvE17Zp1pVtZbcVdThh4"
+  },
+  "12321": {
+    "key": "12321",
+    "name": "Wennect Testnet",
+    "shortName": "Wennect",
+    "testnet": true,
+    "chainId": 12321,
+    "network": "testnet",
+    "multicall": "0x009c55698516Fe3C58105Fe9bBC5A33ECE7A92e4",
+    "rpc": [
+      "https://rpc.testnet.wennect.com"
+    ],
+    "explorer": "https://explorer.testnet.wennect.com",
+    "start": 3606569,
+    "logo": "ipfs://QmTBo3zzrAvJ9p4oLAYtgyEw2LPaKRewHBjAxhWjskhqEU"
```

Comment:
Thanks for the changes @berkcent Looks good now :) except that this image doesn't resolve, make sure to pin this IPFS image (you can use a third-party provider like pinata or fleek or 4everland or others)

Reply:
@ChaituVR changed - please check if it's correct :)

Reply:
I tried multiple times @berkcent but it doesn't work 
https://ipfs.io/ipfs/Qmbn1mAqkDpeEdbfVF61QbidbU9YAaEmAJ9ZaJw9cUbGrRs

It returns this message:
> ipfs resolve -r /ipfs/Qmbn1mAqkDpeEdbfVF61QbidbU9YAaEmAJ9ZaJw9cUbGrRs: invalid path "/ipfs/Qmbn1mAqkDpeEdbfVF61QbidbU9YAaEmAJ9ZaJw9cUbGrRs": invalid CID: selected encoding not supported


Maybe share the image here. i will try to pin it :) make sure it is jpg or png like here https://ipfs.io/ipfs/QmR2UYZczmYa4s8mr9HZHci5AQwyAnwUW7tSUZz7KWF3sA

Reply:
![Original Logo3](https://user-images.githubusercontent.com/39877817/181835061-bab05df1-18e2-48ed-9423-ffdf9364a900.png)
If you could add this, it would be great!